db/migrate/20200330141451_add_status_field_to_projects.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusFieldToProjects < ActiveRecord::Migration[6.0]
+  def change
+    add_column :projects, :status, :string, {null:false, default: ""}
```

Comment:
If not using an enum for this column, we might want to use a validation on the value in the model:

```
  validates :status, inclusion: { in: ALL_PROJECT_STATUS }
```

Reply:
Done